dev-requirements.txt

```diff
@@ -15,3 +15,4 @@ types-PyYAML==6.0.8
 types-redis==4.3.2
 types-toml==0.10.7
 types-ujson==5.2.0
+types-urllib3==1.26.15
```

Comment:
I think urllib3 is a dependency of a dependency because it is installed and mypy installed it when I ran the command to install all needed type stubs.

![Screen Shot 2022-07-05 at 3 09 47 PM](https://user-images.githubusercontent.com/25045024/177398752-e29916f9-5a5c-4a83-bb4e-8569db9aa3ba.png)